pkg/internal/server/grpc/snaphotmetadata_handlers.go

```diff
@@ -17,17 +17,24 @@ limitations under the License.
 package grpc
 
 import (
-	"github.com/kubernetes-csi/external-snapshot-metadata/pkg/api"
+	"fmt"
+	"io"
+
+	"github.com/container-storage-interface/spec/lib/go/csi"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
+
+	"github.com/kubernetes-csi/external-snapshot-metadata/pkg/api"
```

Comment:
No space - these are all in the external import block

Reply:
@carlbraganza "github.com/kubernetes-csi/external-snapshot-metadata/*" are internal packages, so we keep those in separate block. e.g https://github.com/kubernetes-csi/external-snapshot-metadata/blob/a791cfc6a663626ffd4a8cb76c1a6bb88d37d48b/pkg/internal/server/grpc/server.go#L34